portugues/amigo/tecnologia.md

```diff
@@ -0,0 +1,3 @@
+[O seu amigo pede para parar de incomodar toda vez que tem um problema no celular.](incomodar/aplicativo.md)
+
+[O seu amigo gostou muito da aplicação e decidiu lhe ensinar como colaborar com o desenvolvedor.](colabora/colaborar.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.